config/config.go

```diff
@@ -917,9 +917,9 @@ func DefaultMempoolConfig() *MempoolConfig {
 		// Each signature verification takes .5ms, Size reduced until we implement
 		// ABCI Recheck
 		Size:        5000,
-		MaxTxsBytes: 1024 * 1024 * 1024, // 1GB
+		MaxTxBytes:  1024 * 1024,          // 1MB
+		MaxTxsBytes: 16 * 4 * 1024 * 1024, // 64MB
```

Comment:
Curious why not? Trying to show `16 * max_block_size *...`?
```suggestion
		MaxTxsBytes: 64 * 1024 * 1024, // 64MB
```

Reply:
Yes.

Reply:
Agree, ideally if there’s a correlation with these parameters, e.g. tx size, # txs per block, block size, we should look into initializing some values (e.g. MaxTxsBytes = RoundsToDrain * BlockSize * MaxTxBytes) based on a calculation of things that would constrain it

Reply:
Simplified the math, added rationale as a comment

Reply:
Thanks !